.travis.yml

```diff
@@ -36,3 +36,8 @@ jobs:
         - make build-containers || travis_terminate 1;
         - make gen-manifest || travis_terminate 1;
         - tests/travis-test.sh v1.15.0 || travis_terminate 1;
+    - name: kadalu with kube 1.16.0
```

Comment:
can you add Kube 1.13 also?

Reply:
I will add 1.13 as a separate patch. Lets take one at a time :-)

And also, I believe for 1.13.7 we need to add some extra checks, and different files (object/crd) in manifest?

Reply:
nope, nothing is required, current code should work with kube 1.13 as well.

Reply:
Ok, let me try that as a patch on top of this. Let me get to fix this.